src/nvim/edit.c

```diff
@@ -1795,33 +1793,37 @@ void backspace_until_column(int col)
   }
 }
 
-/*
- * Like del_char(), but make sure not to go before column "limit_col".
- * Only matters when there are composing characters.
- * Return TRUE when something was deleted.
- */
-static int del_char_after_col(int limit_col)
+/// Like del_char(), but make sure not to go before column "limit_col".
+/// Only matters when there are composing characters.
+///
+/// @param  limit_col  only delete the character if it is after this column
+//
+/// @return true when something was deleted.
+static bool del_char_after_col(int limit_col)
 {
   if (enc_utf8 && limit_col >= 0) {
     colnr_T ecol = curwin->w_cursor.col + 1;
 
-    /* Make sure the cursor is at the start of a character, but
-     * skip forward again when going too far back because of a
-     * composing character. */
+    // Make sure the cursor is at the start of a character, but
+    // skip forward again when going too far back because of a
+    // composing character.
     mb_adjust_cursor();
     while (curwin->w_cursor.col < (colnr_T)limit_col) {
       int l = utf_ptr2len(get_cursor_pos_ptr());
 
-      if (l == 0)        /* end of line */
+      if (l == 0) {  // end of line
         break;
+      }
       curwin->w_cursor.col += l;
     }
-    if (*get_cursor_pos_ptr() == NUL || curwin->w_cursor.col == ecol)
-      return FALSE;
-    del_bytes((long)((int)ecol - curwin->w_cursor.col), FALSE, TRUE);
-  } else
-    (void)del_char(FALSE);
-  return TRUE;
+    if (*get_cursor_pos_ptr() == NUL || curwin->w_cursor.col == ecol) {
+      return false;
+    }
+    del_bytes((long)((int)ecol - curwin->w_cursor.col), false, true);
+  } else {
+    (void)del_char(false);
```

Comment:
`del_char` is OK/FAIL not bool, so the `(void)` is not needed. But I wonder why this is not `return del_char(false) == OK` which the doc text (return true when something was deleted) seem to indicate.